completions/ping

```diff
@@ -65,8 +65,9 @@ _ping()
     fi
 
     [[ $1 == *6 ]] && ipvx=-6
+    [[ $1 == *4 ]] && ipvx=-4
     _known_hosts_real ${ipvx-} -- "$cur"
 } &&
-    complete -F _ping ping ping6
+    complete -F _ping ping ping4 ping6
```

Comment:
We should add a `ping4` symlink so the completion gets actually loaded for it, similarly as for `ping6`. For reference:
```
$ git grep -wFn ping6 | head -n 3
completions/.gitignore:154:/ping6
completions/Makefile.am:686:	ping6 \
completions/Makefile.am:961:		ping6
```

Reply:
Done.